app/experimenter/experiments/constants.py

```diff
@@ -99,13 +99,15 @@ def FEATURE_TYPE_CHOICES(cls):  # pragma: no cover
     STATUS_ACCEPTED = "Accepted"
     STATUS_LIVE = "Live"
     STATUS_COMPLETE = "Complete"
+    STATUS_REJECTED = "Rejected"
 
     STATUS_DRAFT_LABEL = "Draft"
     STATUS_REVIEW_LABEL = "Ready for Sign-Off"
     STATUS_SHIP_LABEL = "Ready to Ship"
     STATUS_ACCEPTED_LABEL = "Accepted by Normandy"
     STATUS_LIVE_LABEL = "Live"
     STATUS_COMPLETE_LABEL = "Complete"
+    STATUS_REJECTED = "Rejected"
```

Comment:
Q: Would this clobber the previous `STATUS_REJECTED` definition above on L102? Or should this be `STATUS_REJECTED_LABEL` here?

Reply:
oh yup, you're right